uploader/templates/upload.html

```diff
@@ -271,8 +271,8 @@ <h5 class="card-title text-center">
   // Create and render a Picker object for searching images.
   function createPicker() {
     if (pickerApiLoaded && oauthToken) {
-      var view = new google.picker.View(google.picker.ViewId.DOCS);
-      view.setMimeTypes("image/png,image/jpeg,image/jpg");
+      var view = new google.picker.DocsView().setIncludeFolders(true);
+      view.setMimeTypes("");

```

Comment:
Will this result in people uploading non image files ? We should not allow them to see those btw.

Reply:
@tonythomas01 The required changes are done and now the user can only view images and no other files. I request you to review the changes and merge the pull request.
Thanks!